log_utils/remote_logs.py

```diff
@@ -0,0 +1,112 @@
+'''Functions for sending remote logs'''
+
+import base64
+import requests
+
+
+STATUS_CODE_COMPLETE = 0
+STATUS_CODE_ERROR = 1
+
+
+def b64string(text):
+    '''
+    Base64 encodes a string as utf-8
+
+    >>> b64string('boop')
+    'Ym9vcA=='
+    '''
+    return base64.b64encode(text.encode('utf-8')).decode('utf-8')
+
+
+def post_output_log(log_callback_url, source, output, limit=500000,
+                    limit_msg='output suppressed due to length'):
+    '''
+    POSTs an `output` log from `source` to the `log_callback_url`.
+
+    If the `output` length is greater than `limit`, then instead
+    `limit_msg` is sent.
+    '''
+    # Replicates log_output() function in main.sh
+    if len(output) > limit:
+        output = limit_msg
```

Comment:
at some point (in a future PR), if the output length is too long, perhaps we should send a truncated output (i.e., `output[:limit] + '...'`)? it seems like that may be more helpful than simply the `limit_msg`

Reply:
yup, great idea. This was just a port of the existing logic. Opened https://github.com/18F/federalist-garden-build-py/issues/6 to track.